src/components/TodoInput.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function TodoInput(props) {
+  return (
+    <form className="form-styled">
+      <div className="form-group">
```

Comment:
HTML can be simplified here a bit. I think `div.form-group` does not make much sense here. You can remove it and style `form. form-styled` accordingly.